sql-statements/sql-statement-backup.md

```diff
@@ -98,7 +98,7 @@ BACKUP DATABASE * TO 'local:///mnt/backup/full/';
 
 Note that the system tables (`mysql.*`, `INFORMATION_SCHEMA.*`, `PERFORMANCE_SCHEMA.*`, …) will not be included into the backup.
 
-### Remote destinations
+### External storages
 
 BR supports backing up data to S3 or GCS:
 
```

Comment:
Modify the Line 111: BR storages -> external storages

And the same for the Chinese version.

Reply:
Done in https://github.com/pingcap/docs/pull/5286/commits/913a8405f7bbbbae7db13debb64482105afa20db。